docs/features.md

```diff
@@ -27,7 +27,7 @@ Numbers
 
 Ellis manages a database of *numbers*. Each number is either available
 for use, or belongs to a user. Numbers may be internal or PSTN
-numbers, and may or may not be exposed in the global address book.
+numbers, and may or may not be exposed in the global address book. Ellis can't actually provision numbers that can make or receive calls from the PSTN - rather a 'PSTN number' is just a number in international format, e.g. +15108580274.
```

Comment:
I think the phrasing here is off: Ellis **can** provision numbers that can make or receive calls from the PSTN if you have a PSTN trunk. And you'll probably need an international-format number for that to work. (But I bet you could create the non-PSTN number pool in international format, and the PSTN pool in national format...)